modxcms/revolution
# Remove legacy file manager processors [SEC-3837]

### What does it do?

Removes unused legacy filesys processors. 

### Why is it needed?

Reported by Solar Security. These file/folder processors were once part of a file manager, predating media sources with more fine-grained access control. Any manager user with file_manager permission could use these to interact with existing files and create/remove directories.

### How to test

Could test they get removed properly in the setup, and/or confirm the PoC in security report 3837 no longer works because the processors used are gone. 

### Related issue(s)/PR(s)

Private report https://community.modx.com/t/new-security-issues-reported-all-in-one-report/3837/27

(No 3.x port needed, these processors are already removed there.)
